lib/solr/params.rb

```diff
@@ -16,15 +27,19 @@ def initialize
       }
     end
 
-    def fq(field, values, boolean_operator = :or)
-      raise ArgumentError unless [:and, :or].include?(boolean_operator)
-      escaped_values = Array.wrap(values).map { |value| Solr::Utils.escape(value).gsub(' ', '\ ') }
-      @parameters[:fq] << "#{field}:(#{escaped_values.join(boolean_operator == :and ? ' AND ' : ' OR ')})" unless escaped_values.blank?
+    def fq(field, value_or_values, match_operator = 'EQUALS', boolean_operator = :or)
```

Comment:
To confirm: Since this `boolean_operator` param (plus internal method logic) is still here, does that mean that we are still supporting the possibility of passing AND to the `fq` method?  The reason I ask is because that `params.fq('animals', ['dogs + cats', 'llamas', 'alpacas'], :and)` test was deleted and we talked about supporting AND-ing through the addition of multiple, separate fqs.

We should either delete the boolean_operator param or re-add the deleted method to maintain test coverage.
 I'm good to go with whichever way you'd prefer.